seacrowd/sea_datasets/uit_viocd/uit_viocd.py

```diff
@@ -0,0 +1,141 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@incollection{nguyen2021vietnamese,
+  title={Vietnamese Complaint Detection on E-Commerce Websites},
+  author={Nguyen, Nhung Thi-Hong and Ha, Phuong Phan-Dieu and Nguyen, Luan Thanh and Nguyen, Kiet Van and Nguyen, Ngan Luu-Thuy},
+  booktitle={New Trends in Intelligent Software Methodologies, Tools and Techniques},
+  pages={618--629},
+  year={2021},
+  publisher={IOS Press}
+}
+"""
+
+_DATASETNAME = "uit_viocd"
+
+_DESCRIPTION = """\
+The UIT-ViOCD dataset includes 5,485 reviews e-commerce sites across four categories: fashion, cosmetics, applications,
+and phones. Each review is annotated by humans, assigning a label of 1 for complaints and 0 for non-complaints.
+The dataset is divided into training, validation, and test sets, distributed approximately in an 80:10:10 ratio.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/tarudesu/ViOCD"
+
+_LANGUAGES = ["vie"]
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {
+    "train": "https://huggingface.co/datasets/tarudesu/ViOCD/resolve/main/train.csv?download=true",
+    "val": "https://huggingface.co/datasets/tarudesu/ViOCD/resolve/main/val.csv?download=true",
+    "test": "https://huggingface.co/datasets/tarudesu/ViOCD/resolve/main/test.csv?download=true",
+}
+
+_SUPPORTED_TASKS = [Tasks.INTENT_CLASSIFICATION]
```

Comment:
Could you please pull from the `main` branch and change this to `Tasks.COMPLAINT_DETECTION`?